sqliter-driver/src/nativeCommonTest/kotlin/co/touchlab/sqliter/DatabaseConnectionTest.kt

```diff
@@ -276,20 +265,23 @@ class DatabaseConnectionTest {
                 bindString(2, "asdf")
                 executeInsert()
             }
+            it.withStatement("insert into test(num, str)values(?,?)") {
+                bindLong(1, 233)
+                bindString(2, "asdfg")
+                executeInsert()
+            }
```

Comment:
Added a 2nd insertion so that `conn1.size == 1` and `conn2.size == 2` to differentiate between the 2 ephemeral databases created